packages/gatsby/src/query/query-watcher.ts

```diff
@@ -8,34 +8,62 @@
  * - Whenever a query changes, re-run all pages that rely on this query.
  ***/
 
-const chokidar = require(`chokidar`)
-
-const path = require(`path`)
-const { slash } = require(`gatsby-core-utils`)
+import chokidar, { FSWatcher } from "chokidar"
+import { Span } from "opentracing"
+
+import path from "path"
+import { slash } from "gatsby-core-utils"
+
+import { store, emitter } from "../redux/"
+import { boundActionCreators } from "../redux/actions"
+import { IGatsbyStaticQueryComponents } from "../redux/types"
+import queryCompiler from "./query-compiler"
+import report from "gatsby-cli/lib/reporter"
+import queryUtil from "./index"
```

Comment:
```suggestion
import queryUtil from "."
```
Best to avoid directly importing "index"

Reply:
Sure! 4018699